docs/index.rst

```diff
@@ -56,8 +56,9 @@ Step 2: Setup Bioconda
 
 After installing Miniconda, you can use the ``conda`` command to setup bioconda with::
 
-   conda config --add channels r
    conda config --add channels conda-forge
```

Comment:
Will this work? People will have defaults already in their condarc... Don't you need --append?